AngularJS/src/org/angular2/codeInsight/attributes/Angular2AttributeDescriptor.java

```diff
@@ -465,15 +465,27 @@ else if (types.size() > 1) {
   public static boolean isOneTimeBindingProperty(@NotNull Angular2DirectiveProperty property) {
     if (ONE_TIME_BINDING_EXCLUDES.contains(property.getName())) return false;
     if (property.isVirtual()) return true;
-    JSType type = property.getType();
-    if (type == null) return false;
```

Comment:
I think that the only change needed here is to return `true` if the type is `null` to fix the issue. I recall that I purposefully returned `false` here, but I can't recall why 😃 Since `any` type is assignable to `string` this should indeed return `true` here.

Reply:
@lppedd ok! I choosed to manually assign a `JSAnyType` to be sure I didn't lose anything which come after.
I'll do it tonight when I arrive home. Thanks!